### PR TITLE
Added additional information to Customizable options page

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Added additional information to Customizable options page

## Affected documentation pages
https://docs.magento.com/user-guide/catalog/settings-advanced-custom-options.html

### CODE REVIEW

File: src/catalog/settings-advanced-custom-options.md [hunk 1]
```diff
@@ -54,6 +54,8 @@ _Cart with Customizable Option and Price Rule_
 
    - Enter a **SKU** for the option. The option SKU is a suffix that is added to the product SKU.
 
+   - If the `Option Type` is **File**, enter the separated extensions, like: `png`, `jpg`, `gif` into **Compatible File Extensions** field, and image size in px. into **Maximum Image Size** field.
```
Comment (leslietilling) [inline src/catalog/settings-advanced-custom-options.md:57]:
```suggestion
   - If the _Option Type_ is `File`, set the parameters for the file. For **Compatible File Extensions**, enter the valid extensions as comma-separated values (such as `png, jpg, gif`). For **Maximum Image Size**, enter the maximum image size in pixels.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Edits for style and clarity

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@leslietilling the Pull Request is successfully imported.

**Pull request merged**